Compat.c

```diff
@@ -117,3 +123,31 @@ ssize_t Compat_readlinkat(int dirfd,
 
 #endif
 }
+
+int Compat_clock_monotonic_gettime(struct timespec *tp) {
+
+#ifdef HAVE_CLOCK_GETTIME
```

Comment:
What about the following for consistency?
```suggestion
#if defined(HAVE_CLOCK_GETTIME)
```